wordsGeneratorServer/wordsGeneratorServerFinishGame/src/main/kotlin/jetbrains/kotlin/course/words/generator/results/GameResultsService.kt

```diff
@@ -18,5 +18,6 @@ class GameResultsService {
         gameHistory.add(result)
     }
 
-    fun getAllGameResults() = gameHistory.reversed()
+    // It's necessary to specify return type explicitly, so kotlin-reflect gets it right
+    fun getAllGameResults(): List<GameResult> = gameHistory.reversed()
```

Comment:
What if the student does not explicitly specify the return type? I'd suggest adding a check that will handle this case in the tests, WDYT?